spec/features/user_views_posts_with_date_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.feature "Posts", type: :feature do
+  scenario "Can display date with post text" do
+    visit "/posts"
```

Comment:
Could call a reusable test helper to add posts.

Reply:
Thanks, changes made!